configs/machines/aleph.yaml

```diff
@@ -46,53 +46,135 @@ hyper_flag: ""
 #
 #
 
-useMPI: cray_mpich
-
-fc: ftn
-f77: ftn
-mpifc: ftn
-mpicc: cc
-cc: cc
-cxx: CC
-
-module_actions:
-        - "unload craype"
-        - "load craype/2.6.2"
-        - "load PrgEnv-cray/6.0.4"
-        - "load pbs"
-        - "load cray-mpich/7.7.3"
-        - "load craype-x86-skylake"
-        - "load cmake/3.14.0"
-        - "load cray-hdf5-parallel/1.10.2.0"
-        - "load cray-netcdf-hdf5parallel/4.6.1.3"
-        - "load cdo/1.9.5"
-        - "load fftw/2.1.5.9"
-        - "load nco/4.9.4"
-        - "load proj4/5.1.0"
-        - "load python/3.9.1"
-        - "list"
-
+useMPI: cray_mpich #intel_mpich #intelopenmpi #cray_mpich
+
+choose_useMPI:
+        cray_mpich:
+                fc: ftn
+                f77: ftn
+                mpifc: ftn
+                mpicc: cc
+                cc: cc
+                cxx: CC
+                module_actions:
+                        - "unload craype"
+                        - "load craype/2.6.2"
+                        - "load PrgEnv-cray/6.0.4"
+                        - "load pbs"
+                        - "load cray-mpich/7.7.3"
+                        - "load craype-x86-skylake"
+                        - "load cmake/3.14.0"
+                        - "load cray-hdf5-parallel/1.10.2.0"
+                        - "load cray-netcdf-hdf5parallel/4.6.1.3"
+                        - "load cdo/1.9.5"
+                        - "load fftw/2.1.5.9"
+                        - "load nco/4.9.4"
+                        - "load proj4/5.1.0"
+                        - "load python/3.9.1"
+                        - "list"
+                add_export_vars:
+                        ECCODESROOT: /proj/awiiccp/software/ecmwf/eccodes_cce_mpich
+                        HDF5ROOT: $HDF5_ROOT
+
+                        NETCDFFROOT: $NETCDF_DIR
+                        NETCDFROOT: $NETCDF_DIR
+                        NETCDF_Fortran_INCLUDE_DIRECTORIES: $NETCDFROOT/include
+                        NETCDF_CXX_INCLUDE_DIRECTORIES: $NETCDFROOT/include
+                        NETCDF_CXX_LIBRARIES: $NETCDFROOT/lib
+                        ZLIBROOT: /usr
+
+        intelopenmpi:
+                warning:
+                        Configuration does not run: 
+                        message: "The intelopenmpi configuration on aleph is experimental. From past experiance it can compile AWI-CM3, but crashes on runtime. Use as your own disgression. Contact jan.streffing@awi.de for further questions"
+                        ask_user_to_continue: True
```

Comment:
```suggestion
                        "Configuration does not run": 
                            message: "The intelopenmpi configuration on aleph is experimental. From past experiance it can compile AWI-CM3, but crashes on runtime. Use as your own disgression. Contact jan.streffing@awi.de for further questions"
                            ask_user_to_continue: True
```